git-keeper.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# git-keeper utility, used for backing up git repos to AWS S3
+# supported git providers github and gitlab
+# 
+# expecting ssh key for private repos and AWS S3 credentials configured
+# tested only on linux OS
+
+import os
+import sh
+import boto3
+import botocore
+import shutil
+import logging
+from datetime import datetime
+import sys
+import gnupg
+
+# bake some commands
+mirror = sh.git.clone.bake('--mirror')
+tar = sh.tar.bake('-cf')
+
+workdir = 'workdir'
+
+def upload2s3(repo_tar, DATE, object_name):
```

Comment:
instead of recreating the s3_client, can you pass it as an arg here from the main func?
`def upload2s3(s3_client, repo_tar, DATE, object_name):`